source/partials/_nav-version-control-deployments.html.erb

```diff
@@ -1,5 +1,6 @@
 <ul>
   <li><a href="/standards/source-code/">How to store source code</a></li>
+  <li><a href="/standards/source-code/index.html">How to store source code</a></li>
```

Comment:
Isn't this a duplicate of the link above?

Reply:
Yep, will remove